resources/js/src/app/components/itemList/filter/ItemFilter.js

```diff
@@ -63,5 +69,19 @@ Vue.component("findologic-item-filter", {
                 name: parentCategory.name + '_' + subCategory.name
             };
         },
+
+        /**
+         * @returns {boolean}
+         */
+        shouldShowFilter() {
```

Comment:
Can't `shouldShowFilter` and `shouldShowCategoryFilter` be combined somehow?

Reply:
I've moved the checking if facet id is equals to cat to the twig file and left only shouldShowCategoryFilter method in the js component.